airflow/providers/google/cloud/transfers/presto_to_gcs.py

```diff
@@ -129,16 +129,15 @@ def fetchmany(self, size=None) -> list:
     def __next__(self) -> Any:
         """
         Return the next row from the currently executing SQL statement using the same semantics as
-        ``.fetchone()``.  A ``StopIteration`` exception is raised when the result set is exhausted.
-        :return:
+        ``.fetchone()``. A ``StopIteration`` exception is raised when the result set is exhausted.
```

Comment:
Same as above (not sure if there are more)

Reply:
There are not :)